content/en/dev/setup.md

```diff
@@ -49,31 +49,41 @@ You can follow the [pre-requisites instructions from the production guide]({{<re
 Run the following commands in the project directory:
 
 ```sh
+bundle config set --local path vendor/bundle
```

Comment:
is this necessary? i don't remember having to do this

Reply:
I just went through the dev setup instructions myself (hence the drive-by comments, sorry 😅) and didn't have to do this either.

Reply:
Running `bundle install` on Fedora 37 with the `rubygem-bundler` package gives:

![image](https://user-images.githubusercontent.com/107580327/223589483-86c52d1d-e500-43f6-8dea-2e918c62e871.png)

@trwnh @nuztalgia Were you using RVM or rbenv? Perhaps that's why it wasn't necessary

Reply:
don't the prereqs say to use rbenv?

Reply:
@trwnh But the `Vagrantfile` uses RVM :thinking: 

Also, I think rbenv/RVM Bundler still installs the gems globally, it just doesn't require root since rbenv/RVM install Ruby per-user in the home directory. I was thinking that it's better to isolate the gems per-project, which AFAIK rbenv/RVM don't do by default, unless you use [gemsets](https://rvm.io/gemsets/basics) or something?

Reply:
unfortunately i do not know enough about the ruby ecosystem to have an informed opinion on this ^_^; but i guess it might be worth making a more general statement about how dependencies have to be installed in a non-root way, and then suggest using one of rbenv, rvm, or setting the bundle local path. or, based on some quick googling, i see that other people might recommend `chruby` for this? not really sure if any of these approaches are "best", or what the industry standards might be

Reply:
@trwnh Does ddfd3e82d64e723a5f248e750deddf949c5fadcc look OK? Thanks for your help by the way :smile: